sources/packages/web/src/components/layouts/BodyHeaderContainer.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <!-- Header slot (with and without v-card) -->
+    <v-card v-if="card"><slot name="header"></slot></v-card>
+    <div v-else><slot name="header"></slot></div>
+
+    <!-- Default slot (with and without v-card) -->
+    <v-card v-if="card" class="mt-n1"><slot></slot></v-card>
```

Comment:
if `card==true` then will there be one `v-card` for the header and another `v-card` for the body?

Reply:
Yes, when card (enableCardView) is set to true, there is a v-card for the header and another one for the body.

Reply:
Fixed it.